### PR TITLE
Add help option to FormType

### PR DESCRIPTION
**Pull request by mpiot**
Add documentation about the new feature symfony/symfony#26332

### CODE REVIEW

File: reference/forms/types/options/help.rst.inc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+help
+~~~~
+
+**type**: ``string`` **default**: null
+
+Allows you to define a help text below the form field.
```
Comment (wouterj) [inline reference/forms/types/options/help.rst.inc]: What about something like: "Allows you to define extra information about the field, which will be rendered below the form field by default." (as you're allowed to change the layout of a form in every way you want)

File: reference/forms/types/options/help.rst.inc [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+    $builder->add('name', null, array(
+    'help'   => 'My helper message.',
+    ));
```
Comment (wouterj) [inline reference/forms/types/options/help.rst.inc]: Please indent the array entry with 4 spaces and you have to add `.. code-block:: php` before this code example in order to render it as PHP code.

File: reference/twig_reference.rst [hunk 3]
```diff
@@ -232,6 +232,18 @@ form_label
...
+~~~~~~~~~~
+
+.. code-block:: twig
+
+    {{ form_help(view) }}
+
+``view``
+    **type**: ``FormView``
+
+Renders the text help for the given field.
```
Comment (wouterj) [inline reference/twig_reference.rst]: "Renders the text help" -> "Renders the help text" ?

**Review by wouterj — CHANGES_REQUESTED** [review]
Looks like a great feature & very complete doc PR! I've added some small comments, would be awesome if you can manage to fix them.

File: reference/twig_reference.rst (hunk 3 above)
Comment (mpiot) [reply reference/twig_reference.rst]: I've fixed it too, but apparently, don't recognized.

File: reference/forms/types/birthday.rst [hunk 4]
```diff
@@ -42,6 +42,7 @@ option defaults to 120 years ago to the current year.
 |                      | - `invalid_message`_                                                          |
 |                      | - `invalid_message_parameters`_                                               |
 |                      | - `mapped`_                                                                   |
+|                      | - `help`_                                                                     |
```
Comment (xabbuh) [inline reference/forms/types/birthday.rst]: should be between `invalid_message_parameters` and `mapped` IMO to keep options in alphabetical order

Comment (xabbuh) [reply reference/forms/types/birthday.rst]: same in the other files

**Review by ndench — APPROVED** [review]
Looks good to me, except for the comment by @xabbuh about alphabetical ordering. Thanks for the contribution, I was looking for this exact documentation just the other day.

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@mpiot thanks for providing these docs and congrats on your first Symfony Docs contribution. Also, thanks for contributing this great feature to Symfony forms.

**Pull request closed without merging**